lib/rubocop/rspec_rails/language.rb

```diff
@@ -13,7 +13,6 @@ module RSpec
     # using the configured aliases.
     module Language
```

Comment:
Won’t it be too confusing to have both rubocop-rspec and rubocop-rspec_rails define a `Language` module. Would it be feasible to include `RuboCop::RSpec::Language` into the new Base class?

Or wait… maybe it’s just the `Language` configuration in config/default.yml I have objections to.

Reply:
I see. I was conversing with @pirj in the following thread, and I don't think we should depends on rubocop-rspec. duplicating Language is redundant, but I think it's necessary for each to be independent. I wonder if the reason it doesn't feel right is because the class name is RSpecRails::Language?

https://github.com/ydah/rubocop-rspec_rails/pull/1#discussion_r1520037079